client/signup/style.scss

```diff
@@ -425,3 +425,77 @@ body.is-section-signup.is-new-launch-flow .layout:not( .dops ):not( .is-wccom-oa
 		color: var( --color-text );
 	}
 }
+
+/**
+ * Common styles for reskinSignupFlow a/b test
+ */
+body.is-section-signup.is-white-signup .signup.is-onboarding {
+	$gray-100: #101517;
+	$gray-60: #50575e;
+	$gray-50: #646970;
+	$breakpoint-mobile: 660px;
+
+	--color-accent: #117ac9;
```

Comment:
Thanks for adding this!
I talked to the design team, we would also need to add
```css
--color-accent-60: #0e64a5;
```
As this is the color used for the hover states of buttons.